ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/util/MetadataUtil.java

```diff
@@ -47,4 +55,105 @@ public static String getCurrentVersion(final MigrationConfig config, final Clien
           String.format("Could not query %s: %s", migrationTableName, e.getMessage()));
     }
   }
+
+  public static String getLatestMigratedVersion(
```

Comment:
@jzaralim this method will be useful for implementing `apply`, in order to get the latest version to start applying commands from.